src/axom/slam/IndirectionSet.hpp

```diff
@@ -22,21 +22,21 @@ namespace axom
 namespace slam
 {
 /**
- * \brief Alias template for an OrderedSet with indirection over an array
+ * \brief Alias template for an OrderedSet with indirection over a C array
  *
  * \tparam PosType The position type for indexing into the set
  * \tparam ElemType The type for the set's elements
  * \sa OrderedSet
  */
 template <typename PosType = slam::DefaultPositionType,
           typename ElemType = slam::DefaultElementType>
-using ArrayIndirectionSet =
+using CArrayIndirectionSet =
```

Comment:
@kanye-quest -- I renamed some of the indirection policy classes.
* for C-style arrays, `ArrayIndirection` seemed confusing now that we have `axom::Array`. I renamed it to `CArrayIndirection`
* similarly, `CoreArrayIndirection` (below) seemed a bit obscure. I renamed it to `ArrayIndirection` to better match `ArrayViewIndirection`.

I added a note about this to the RELEASE-NOTES. Please let me know what you think.